app/main.py

```diff
@@ -1 +1,26 @@
-# write your code here
+from os import makedirs, remove, rename
+
+
+def move_file(command: str) -> None:
+    com, old_filename, path_and_new_filename = command.split()
+    new_filename = path_and_new_filename.split("/")[-1]
```

Comment:
Each OS has difference path separator, so you can't use `/` split. Use `os.path.split` instead of `.split("/")`